src/containers/Tenant/utils/schemaQueryTemplates.ts

```diff
@@ -54,66 +60,79 @@ PARTITION BY HASH(id)
 WITH (STORE = COLUMN)`;
 };
 export const createAsyncReplicationTemplate = () => {
-    return `CREATE OBJECT secret_name (TYPE SECRET) WITH value="secret_value";
+    return `-- docs: https://ydb.tech/docs/en/yql/reference/syntax/create-async-replication
+CREATE OBJECT secret_name (TYPE SECRET) WITH value="secret_value";
 
 CREATE ASYNC REPLICATION my_replication
-FOR \`/remote_database/table_name\` AS \`local_table_name\` --[, \`/remote_database/another_table_name\` AS \`another_local_table_name\` ...]
+FOR \${1:<original_table>} AS \${2:replica_table} --[, \`/remote_database/another_table_name\` AS \`another_local_table_name\` ...]
 WITH (
-    CONNECTION_STRING="grpcs://mydb.ydb.tech:2135/?database=/remote_database",
+    CONNECTION_STRING="grpcs://mydb.ydb.tech:2135/?database=/\${3:<remote_database>}",
     TOKEN_SECRET_NAME = "secret_name"
     -- ENDPOINT="mydb.ydb.tech:2135",
-    -- DATABASE=\`/remote_database\`,
+    -- DATABASE=\`\${3:/remote_database}\`,
     -- USER="user",
     -- PASSWORD_SECRET_NAME="your_password"
 );`;
 };
 export const alterTableTemplate = (params?: SchemaQueryParams) => {
+    const path = params?.relativePath ? `\`${params?.relativePath}\`` : '${1:<my_table>}';
+
     return `-- docs: https://ydb.tech/docs/en/yql/reference/syntax/alter_table/
 
-ALTER TABLE \`${params?.relativePath || '$path'}\`
+ALTER TABLE ${path}
     -- RENAME TO new_table_name
     -- DROP COLUMN some_existing_column
-    ADD COLUMN numeric_column Int32;`;
+\${2:ADD COLUMN numeric_column Int32};`;
 };
 export const selectQueryTemplate = (params?: SchemaQueryParams) => {
-    const columns = params?.tableData?.map((column) => '`' + column.name + '`').join(', ') || '*';
+    const path = params?.relativePath ? `\`${params?.relativePath}\`` : '${2:<my_table>}';
+    const columns =
+        params?.tableData?.map((column) => '`' + column.name + '`').join(', ') || '${1:*}';
 
     return `SELECT ${columns}
-    FROM \`${params?.relativePath || '$path'}\`
-    LIMIT 10;`;
+FROM ${path}
+WHERE \${3:Key1 = 1}
+ORDER BY \${4:Key1}
+LIMIT \${5:10};`;
```

Comment:
Since we use snippets, it's better to change order to enable columns autocomplete when filling snippets

`FROM table SELECT columns where....`

Reply:
discussed with @adameat and he insisted to keep present order in query